tests/test_pyproject.py

```diff
@@ -37,6 +44,23 @@ def test_python_project_metadata_detect(project_dir, expected):
     assert lookup_metadata_file(project_dir) == expectation
 
 
+@pytest.mark.parametrize(
```

Comment:
Maybe consider a comment describing this test?

Reply:
There is one at line 58, do think it's not sufficiently informative?

Reply:
Yes, sorry, I caught myself seeing that for a different test and suppressed my comment. This is fine.